packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -372,6 +379,34 @@ export default function PostOptionsMenu({
     }
   }
 
+  if (isLoggedIn && post?.author) {
+    const authorName = post.author.name || post.author.username;
+    const isFollowingUser = [
+      ContentPreferenceStatus.Follow,
+      ContentPreferenceStatus.Subscribed,
```

Comment:
Nothing urgent, but maybe we need to consolidate this "fn" at some point.
Pretty sure we are going to need it elsewhere.

Reply:
Added util 6f1d155476649456dea14214d4298990d12160d9